Backend/src/modules/account/tests/create_transfer.rs

```diff
@@ -463,3 +463,36 @@ fn create_account_nick_malformed_email_used_password_valid() {
     // Then
     assert_eq!(response_used.status(), Status::new(522, "InvalidNickname"));
 }
+
+#[test]
+fn invalid_password_characters() {
+    let container = TestContainer::new(false);
+    let (dns, _node) = container.run();
+
+    // Given
+    let http_client = create_http_client(dns.clone());
+    // An object that contains the input parameters is defined
+    let post_obj = CreateMember {
+        nickname: "someNickname".to_string(),
+        credentials: Credentials {
+            mail: "someEmail@someDomain.test".to_string(),
+            password: "%$&§%".to_string(),
```

Comment:
Currently only alphanumeric characters (letters a-z, A-Z and numbers 0-9) are allowed. I think this is way too restrictive.

Reply:
I agree